javaee/impl/src/main/java/org/jboss/forge/addon/javaee/cdi/ui/BeanScope.java

```diff
@@ -11,22 +11,28 @@
  */
 public enum BeanScope
 {
-   DEPENDENT(""),
-   APPLICATION("javax.enterprise.context.ApplicationScoped"),
-   SESSION("javax.enterprise.context.SessionScoped"),
-   CONVERSATION("javax.enterprise.context.ConversationScoped"),
-   REQUEST("javax.enterprise.context.RequestScoped"),
-   CUSTOM(null);
+   DEPENDENT("", false),
```

Comment:
I realized that when you have a session, application or conversation scope bean, it must be serializable.